spec/support/document_crud.rb

```diff
@@ -0,0 +1,22 @@
+module DocumentCrud
+
+  def document_create(params)
```

Comment:
hmm.. not sure if I would put it there. Since other specs may be using this too, I think its more appropriate for it to be called with the correct params. Such as document_create(valid_params.merge(_id: 'its_a_dupe')).

Reply:
Yeah, that's what makes me nervous. The person writing the spec has to know in advance that the params passed to `document_create` have to use the `_id` param, rather than the `document_id` param. But let's leave it as is for now - we can consider adding some automagicness if that does appear to be a problem.